tests/testthat/test-yoda_test_linter.R

```diff
@@ -37,10 +37,15 @@ test_that("yoda_test_linter ignores strings in $ expressions", {
 })
 
 # if we only inspect the first argument & ignore context, get false positives
-test_that("yoda_test_linter ignores usage in pipelines", {
-  expect_lint("foo() %>% expect_identical(2)", NULL, yoda_test_linter())
-  skip_if_not_r_version("4.1.0")
-  expect_lint("bar() |> expect_equal('a')", NULL, yoda_test_linter())
+local({
+  pipes <- c("%>%", "%!>%", "%T>%", "%<>%", if (getRversion() >= "4.1.0") "|>")
```

Comment:
backpipe makes no sense here.

Reply:
pretty baroque for sure... is there a cost to keeping it in?

Reply:
I'm just not sure it's worth testing such a piece of code that couldn't be functional even in theory.
That said, it's probably slight overkill to test all pipes anyway.

Reply:
I see what you mean more on second look, at first I convinced myself we could come up with an example that works.

But such an example would have to be like

```r
x <- "a"
"a" %<>% expect_equal(x)
expect_equal(a, "a")
```

Very peculiar.

Reply:
Removed `%<>%` from consideration for `yoda_test_linter()`